tests/web/test_main.py

```diff
@@ -439,8 +440,27 @@ def test_fetchdf(web_sushi_context: Context) -> None:
 
 def test_get_models(web_sushi_context: Context) -> None:
     response = client.get("/api/models")
+    json_models = [
```

Comment:
The original test wasn't very comprehensive. I think it just checks all models are found in the response from the API. There's not much logic in the endpoint, but the test could definitely be improved. However, I don't think this tests much since the code to generate the expected output looks almost identical to the code that generates the actual output.